### DISCUSSION TITLE
Feature Request: Higher content density in generate command

### DISCUSSION THREAD

**Discussion by jeffbcross** [nrwl/nx-console#1022] [Feature Requests]
Right now, I have to scroll too much to see all the available options when running the generate command, and I sometimes want to glance back and forth at different options before I execute. 

Each option takes up roughly 115 pixels of vertical space. I'd personally prefer if it were less than half that.

![image](https://user-images.githubusercontent.com/463703/108544480-c11bea80-72a3-11eb-9ccb-16c96debba2a.png)

Another way to help me scan all options would be to display the values I've already set, perhaps in the right-side gutter. Like:
```
name: Jeff's Component
project: jeffs-design-system
```

**Comment by sandikbarr** [comment]
@jeffbcross what are your thoughts on this as far as showing the values that have already been set in the form:
* On the left side fixed menu, color the property name in the menu to indicate that it's been modified. From this menu, users can click to the fields to review settings.
* To the right of the form below the command in the header, display the list of properties that have set formatted as arguments to the command 
* The form itself already highlights modified fields with the blue left border
* (Margin-bottom on form fields in this has also been reduced to 1rem/16px)
![Screen Shot 2021-04-01 at 4 35 40 PM](https://user-images.githubusercontent.com/2250413/113356747-68128e00-9308-11eb-8ef2-cfeeca9600dd.png)
